src/Uno.AzureDevOps/Uno.AzureDevOps.Views/App.xaml.cs

```diff
@@ -29,6 +29,8 @@ namespace Uno.AzureDevOps
 	/// </summary>
 	public sealed partial class App : Application
 	{
+		public static bool LoggedOut;
```

Comment:
Don't put a state on the global App. Need to be in a service.

Reply:
Put in in this PR as it has been requested (to avoid two PRs ...)